static/scss/sidebar.scss

```diff
@@ -1,4 +1,8 @@
 .sidebar {
+  // this ensures the sidebar is visible when opaque-overlay covers the screen
+  z-index: 22;
```

Comment:
Take or leave: you could use a variable for the sidebar z-index, then do something like `$sidebarZIndex-1` for the overlay z-index value